app/scss/_print.scss

```diff
@@ -65,6 +72,7 @@
     pre, blockquote {
         border: 0;
         border-left: 3px solid $silver;
+        margin-bottom: 11pt !important;
```

Comment:
Because it is the unit used for paragraphs, and I wanted a one-(paragraph-)line margin.